syntax = "proto3";

package example.maps;

option go_package = "github.com/NenadPantelic/proto-go/proto";

message IdWrapper {
  uint32 id = 1;
}

message MapExample {
  map<string, IdWrapper> ids = 1;
}